ontology/uberon.md

```diff
@@ -178,5 +182,4 @@ wikidata_template: https://en.wikipedia.org/wiki/Template:Uberon
 activity_status: active
 ---
 
-Uberon is an integrated cross-species ontology covering anatomical structures in animals. See the <a href="http://uberon.org">Uberon website</a> for more info, or read the <a
```

Comment:
note that the link to the website and the publication are now redundant of the structured information contained within the frontmatter